sklearn/manifold/t_sne.py

```diff
@@ -191,7 +192,7 @@ def _kl_divergence(params, P, degrees_of_freedom, n_samples, n_components,
 
 def _kl_divergence_bh(params, P, degrees_of_freedom, n_samples, n_components,
                       angle=0.5, skip_num_points=0, verbose=False,
-                      compute_error=True):
+                      compute_error=True, n_jobs=1):
```

Comment:
I'd set the default to None instead and call `effective_n_jobs` within the function